jekyll/_cci2/custom-images.md

```diff
@@ -20,6 +20,10 @@ Watch the following video for a detailed tutorial of customizing Docker images.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/JYVLeguIbe0" frameborder="0" allowfullscreen></iframe>
 
+## How to Use the CircleCI Dockerfile Wizard
+
+Refer to the [`dockerfile-wizard` GitHub repository of CircleCI Public](https://github.com/circleci-public/dockerfile-wizard) for instructions to clone and use the wizard to create a Dockerfile to generate your custom image. 
```

Comment:
Even if we keep the above heading, I'd move all of this beneath the Custom Image Overview since the docker image coming out of the wizard is _still_ a custom image — just with more handholding and less manual input.

Reply:
Hmm, yeah, I thought about that too